index.html

```diff
@@ -0,0 +1,191 @@
+<html>
+<head> 
+  <title> </title>
+ <link rel = "stylesheet" href="css/style.css" />
+</head>
+<body>
+<nav id="navbar"> 
+    <header> JS Documentation </header>
+    <ul>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li><a class="nav-link" href="#What_you_should_know_already">What you should know already</a></li>
+        <li><a class="nav-link" href="#Javascript_and_Java">Javascript and Java</a></li>
+        <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+        <li><a class="nav-link" href="#Variables">Variables</a></li>
+        <li><a class="nav-link" href="#Declaring_variables">Declaring variables</a></li>
+        <li><a class="nav-link" href="#Variables_scope">Variables scope</a></li>
+        <li><a class="nav-link" href="#Global_variables">Global variables</a></li>
+        <li><a class="nav-link" href="#Constants">Constants</a></li>
+        <li><a class="nav-link" href="#Data_types">Data types</a></li>
+        <li><a class="nav-link" href="#if...else_statements">if...else statements scope</a></li>
+        <li><a class="nav-link" href="#while_statements">while statements</a></li>
+        <li><a class="nav-link" href="#Function_declaration">Function declaration</a></li>
+        <li><a class="nav-link" href="#Reference">Reference</a></li>
+       
+    </ul>
+</nav>
+
+<main id="main-doc">
+  <section class="main-section" id="Introduction"> 
+    <header>
+      Introduction
+    </header>
+    <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a host environment (for example, a web browser), JavaScript can be connected 
+        to the objects of its environment to provide programmatic control over them.</p>
+  </section>
+  <section class="main-section"id="What_you_should_know_already" >
+    <header>
+      What you should know already
+    </header>
+    <p>This guide assumes you have the following basic background:</p>
+    <ul>
+        <li>General understanding of the Internet and the World Wide Web (WWW).</li>
+        <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+    </ul>
+  </section>
+  <section class="main-section" id="Javascript_and_Java"> 
+    <header>
+      JavaScript and Java
+    </header>
```

Comment:
✔️ Good job done using Ids. But it seems your code has so many ids. 🤔 I think just one or two is okay for a page. Many ids can create problems for managing styles in one place
id and header issues

✔️ Good semantics job by using header. But some headers seems to have no style class. 🤔 I think If you have others pages where headers change it will be difficult to manage. Kindly work on it.
id and header issues2